From: uazo <uazo@users.noreply.github.com>
Date: Thu, 30 May 2024 11:42:21 +0000
Subject: Media fingerprinting mitigations

Removes the ability to understand the capabilities of the video card in windows.
See https://github.com/uazo/cromite/issues/862

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 third_party/blink/renderer/modules/webcodecs/video_decoder.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/modules/webcodecs/video_decoder.cc b/third_party/blink/renderer/modules/webcodecs/video_decoder.cc
--- a/third_party/blink/renderer/modules/webcodecs/video_decoder.cc
+++ b/third_party/blink/renderer/modules/webcodecs/video_decoder.cc
@@ -305,7 +305,7 @@ ScriptPromise<VideoDecoderSupport> VideoDecoder::isConfigSupported(
   auto* config_copy = CopyConfig(*config);
 
   // Run the "Check Configuration Support" algorithm.
-  HardwarePreference hw_pref = GetHardwareAccelerationPreference(*config_copy);
+  HardwarePreference hw_pref = HardwarePreference::kPreferSoftware;
   VideoDecoderSupport* support = VideoDecoderSupport::Create();
   support->setConfig(config_copy);
 
--
